reactive/filebeat.py

```diff
@@ -193,6 +197,35 @@ def check_filebeat_repo():
     remove_state("filebeat.repo.changed")
 
 
+@when("config.changed")
```

Comment:
[non-blocking] I guess this is also not required. The beat-layers](https://github.com/canonical/layer-beats-base/blob/master/reactive/beats_base.py#L9-L12) has `config-changed` that `set_state("beat.render")`, and this trigger `render_filebeat_template()`

So this mean, `is_config_valid` can be a simple python function that check if the config is valid. Then, in the `render_filebeat_template()` we can have

```python
if not is_config_valid():
   status.blocked("Invalid config. Please check the logs for details.")
```

Reply:
fixed, thanks